Ginger/GingerCoreNET/WorkSpaceLib/WorkSpace.cs

```diff
@@ -384,23 +390,28 @@ public bool OpenSolution(string solutionFolder)
                 SolutionUpgrade.ClearPreviousScans();
                 if (SolutionUpgrade.IsGingerUpgradeNeeded(solutionFolder, solutionFiles))
                 {
+                    Console.WriteLine("Error: Current Ginger version");
                     Reporter.ToLog(eLogLevel.WARN, "Loading Solution- Error: Current Ginger version can't load the Solution because it includes items from higher Ginger version");
                     return false;
                 }
-
+                
                 Reporter.ToLog(eLogLevel.DEBUG, "Loading Solution- Loading Solution xml into object");
                 Solution solution = Solution.LoadSolution(solutionFile);
+                Console.WriteLine("Solution Laaded Successfully");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4707297)